_posts/2022-07-07-premature-dispsosal.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Premature disposal
```

Comment:
not super happy about the title but can't come up with a better one :/

Reply:
- The case of premature disposal 
- Hold on! You're disposing too fast
- Dispose it, but not so quickly!

Reply:
* Is `using var` dangerous?
* syntactic sugar crash
* dispose after me
* async/await pitfalls: premature disposal